strings/java_substring/Solution.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+import java.lang.String;
+
+public class Solution {
+  
+  private static String findSubstring(String wordToCut, int startIndex, int endIndex) {
+    return wordToCut.substring(startIndex, endIndex);
+  }
+
+  public static void main(String[] args) {
+    Scanner in = new Scanner(System.in);
+    String word = in.next();
```

Comment:
It's not guaranteed to be a word, could be a sentence of multiple words or a string without letters at all. Suggested name: line.